### PR TITLE
Andy/ops 11026 working days

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: app/assets/stylesheets/self-service-time-to-pay.css [hunk 1]
```diff
@@ -242,10 +242,10 @@ input.accountName{
     height: 1.9em;
 }
 @media (min-width: 48.0625em){
-    #id_summary ..govuk-summary-list__key {
+    #id_summary .govuk-summary-list__key {
```
Comment (AndyHWChung) [inline app/assets/stylesheets/self-service-time-to-pay.css:245]: intellij said this was a syntax error - is this ok now @tp10?

File: test/testsupport/ItSpec.scala [hunk 2]
```diff
@@ -56,7 +59,9 @@ class ItSpec
   protected val overrideConfig: Map[String, Any] = Map.empty
 
   protected lazy val configMap: Map[String, Any] = Map[String, Any](
+    "mongodb.uri" -> "mongodb://localhost:27017/self-service-time-to-pay-frontend-tests",
```
Comment (AndyHWChung) [inline test/testsupport/ItSpec.scala:62]: I've made the tests use a separate database then the one used by the actual local instance - I found the collections I had were bloated with a load of journeys which I hadn't actually gone through locally myself. I made the test clean up the data it creates as well down below.

File: test/ssttpcalculator/legacy/LegacyCalculatorServiceSpec.scala [hunk 3]
```diff
@@ -44,207 +44,145 @@ class LegacyCalculatorServiceSpec extends ItSpec with Matchers with DateSupport
...
 
-      makeCalculatorInputForPayToday(debits)(clock) shouldBe TaxPaymentPlan(
+      makeCalculatorInputForPayToday(debits, addWorkingDaysResult(clock, june(_8th)))(clock) shouldBe TaxPaymentPlan(
         debits, noInitialPayment, startDate = may(_29th), endDate = june(_30th), firstPaymentDate = Some(july(_1st)))
     }
   }
 
   "paymentScheduleRequest should" - {
     "return a payment schedule request without an initial payment when" - {
-      "the current date is Friday 1st May with upcoming bank holiday" in {
```
Comment (AndyHWChung) [inline test/ssttpcalculator/legacy/LegacyCalculatorServiceSpec.scala:82]: we don't handle the working day calculation directly in this service anymore so I removed a bunch of tests around bank holiday stuff

File: app/assets/stylesheets/self-service-time-to-pay.css (hunk 1 above)
Comment (tp10) [reply app/assets/stylesheets/self-service-time-to-pay.css:245]: Yes, a double dot is a syntax error.  The new version is valid CSS.

**Review by Kntanos — APPROVED** [review]

**Pull request merged**